proposed/event-loop-meta.md

```diff
@@ -33,3 +33,7 @@ The functionality exposed by this interface should include the ability to:
 - Run single and periodic timers
 - Listen for signals
 - Defer execution
+
+## 3.2 Non-Goals
+
+- Specify stream, promise or coroutine interfaces. They don't in the scope of an event loop but in their own respective PSRs.
```

Comment:
Not sure if it's worth splitting these out onto separate bullet points (as they are separate `non-goals`).

The second sentence is missing a word (I think it should be: _"They don't **lie** in the scope..."_) - it might be worth bringing this out to the top.

Something like this: ?

```markdown
The following concepts are not considered to be within the scope of this event loop proposal, and should be covered by separate standardisations:

- Streams
- Promises
- Coroutines
```